pkg/command/completion_helper.go

```diff
@@ -11,9 +11,13 @@ const (
 	// Completion strings for the values of the --target flag
 	compGlobalTarget = "global\tApplicable globally"
 	compK8sTarget    = "k8s\tFor interactions with a Kubernetes cluster"
-	compTAETarget    = "tae\tFor interactions with a Application Engine endpoint"
 	compTMCTarget    = "tmc\tFor interactions with a Mission-Control endpoint"
 
+	// Completion strings for the values of the --type flag
+	compK8sContextType = "k8s\tContext for a Kubernetes cluster"
+	compTAEContextType = "tae\tContext for a Tanzu Application Engine endpoint"
+	compTMCContextType = "tmc\tContext for a Tanzu Mission Control endpoint"
```

Comment:
Here you've added the term `Tanzu`.  I'm not against it but if you want to call it `Tanzu Mission Control endpoint` can you use the same term for `compTMCTarget` right above?

Reply:
Yeah. I will update the `compTMCTarget` to include `Tanzu`. The reason behind it is we are using short forms like `tae` and `tmc`. So, I feel it might be better to include the `Tanzu` prefix to it so that users can relate it to the shorter forms easily.

Reply:
Good point 👍